common/config/sizeauditor/sizeauditor.json

```diff
@@ -0,0 +1,6 @@
+{
+  "sizeTolerance": "1024",
+  "trackingEntity": "Asset",
+  "devopsDropFolderName": "drop",
+  "devopsWebpackStatsArtifactName": "bundlesizes.json"
```

Comment:
Would a path for the above fields `devopsDropFolderName` and `devopsWebpackStatsArtifactName` be preferable?

Reply:
Artifacts for a certain pipeline are required to be unique in name, and we download artifacts using this API
https://docs.microsoft.com/en-us/rest/api/azure/devops/build/artifacts/get?view=azure-devops-rest-4.1

In the case of `office-ui-fabric-react`, that would be similar to
https://dev.azure.com/uifabric/fabricpublic/_apis/build/builds/1234/artifacts?artifactName=drop&api-version=4.1

We only need the name of the artifact (**drop** in this case), don't think we can supply a path, and `bundlesizes.json` is the name of the file inside `drop` that contains all the size information output by the `bundle-size-collect` script https://github.com/OfficeDev/office-ui-fabric-react/blob/cc75f70e933ce6cfb459ec3931ef274abe87ddf3/scripts/tasks/bundle-size-collect.js#L16